notebooks/Media-Pretrained/01_Data_Layer.ipynb

```diff
@@ -9,20 +9,12 @@
     "In this notebook, you will choose a dataset and prepare it for use with Amazon Personalize.\n",
     "\n",
     "1. [How to Use the Notebook](#usenotebook)\n",
-    "1. [Introduction](#intro)\n",
     "1. [Define your Use Case](#usecase)\n",
     "1. [Choose a Dataset or Data Source](#source)\n",
     "1. [Prepare the Item Metadata](#prepare_items)\n",
     "1. [Prepare the Interactions Data](#prepare_interactions)\n",
     "1. [Prepare the User Metadata](#prepare_users)\n",
-    "1. [Configure an S3 bucket and an IAM  role](#bucket_role)\n",
-    "1. [Create Dataset Group](#group_dataset)\n",
-    "1. [Create the Interactions Schema](#interact_schema)\n",
-    "1. [Create the Items (Movies) Schema](#items_schema)\n",
-    "1. [Create the Users Schema](#users_schema)\n",
-    "1. [Import the Interactions Data](#import_interactions)\n",
-    "1. [Import the Item Metadata](#import_items)\n",
-    "1. [Import the User Metadata](#import_users)\n",
+    "\n",
```

Comment:
General: not part of this PR, but there are several instances of this spelling error in this notebook:

Top **pics** for you